pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd_migration_use-cases/guide.de-de.md

```diff
@@ -38,79 +43,41 @@ details[open]>summary::before {
+| **Step 2**  | ❎           | **FT (fault tolerance)**                 | **Disable FT from VMs in managed vSphere on OVHcloud.**    |                                                                                                                                                                                                                        | [VMware Fault Tolerance](/pages/bare_metal_cloud/managed_bare_metal/vmware_fault_tolerance)                                                                                     |                                                                                                                                                                                                                                                                                                                                                            
+| **Step 3**  | ️❎          | **DRS affinity/anti-affinity**           | **Reconstitution of affinity/anti-affinity rules in VCD.** | - Between VMs migration can be migrated to VCD, but between VMs and hosts cannot be migrated.                                                                                                                          | [VMware DRS distributed resource scheduler](/pages/bare_metal_cloud/managed_bare_metal/vmware_drs_distributed_resource_scheduler)                                               |                                                                                                                                                                                                                                                                                                                                               
+| **Step 4**  | ❎           | **Special devices (CD, DVD etc..)**      | **Unplug all special devices.**                            | - Nothing must be plugged into to migrated vSphere managed data centers. All special devices (CD, DVD etc..) must be removed before migration, otherwise they will be force-removed by the migration process and lost. |                                                                                                                                                                                 |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               
```

Comment:
"- Nothing must be plugged into to migrated vSphere managed data centers."

Do you mean on VMs?